lib/src/widgets/ds_bottomsheet_countries.widget.dart

```diff
@@ -0,0 +1,400 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:get/get.dart';
+import '../../blip_ds.dart';
+import '../models/ds_country.model.dart';
+import 'fields/ds_app_search_input.widget.dart';
+
+abstract class DSBottomSheetCountries {
+  static final showClearButton = RxBool(false);
+  static final _filterCountries = RxList<DSCountry>([]);
```

Comment:
You should remove the ```[]``` initializer, since that is already the default one.

Reply:
Done.